server/schedulers/balance_test.go

```diff
@@ -11,179 +11,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package server
+package schedulers
 
 import (
-	"math"
-	"time"
-
 	. "github.com/pingcap/check"
-	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/kvproto/pkg/pdpb"
 	"github.com/pingcap/pd/server/cache"
 	"github.com/pingcap/pd/server/core"
 	"github.com/pingcap/pd/server/namespace"
 	"github.com/pingcap/pd/server/schedule"
-	_ "github.com/pingcap/pd/server/schedulers" // Register schedulers for tests.
+	"math"
```

Comment:
We usually put builtin packages before 3rd-part ones.

Reply:
done